components/Article.jsx

```diff
@@ -0,0 +1,38 @@
+import {Link} from "react-router-dom";
+
+export const Article = (article) => {
+    const {author, title, article_id, topic, created_at, votes, article_img_url, comment_count, body} = article.article;
+
+    const formatDate = (dateString) => {
+        const options = {year: "numeric", month: "long", day: "numeric"};
+        return new Date(dateString).toLocaleDateString(undefined, options);
+    };
+
+    return (
+        <article className="flex flex-col">
+            <section className="flex flex-col md:flex-row md:pb-4">
+                <figure className="w-full md:w-1/2">
+                    <img src={article_img_url} alt={title} className="w-full h-auto rounded-md"/>
+                </figure>
+                <summary className="md:pl-4 flex flex-col justify-center">
+                    <Link to={`/articles/${article_id}`} className="text-2xl font-bold hover:underline">{title}</Link>
+                    <summary className="text-sm hidden md:block py-1">
+                        <Link to={`/topics/${topic}`} className="text-gray-700 hover:underline">
+                            {topic}
+                        </Link>
+                    </summary>
+                    <time className="text-gray-700 text-sm mb-2">
+                        {formatDate(created_at)} - {author}
+                    </time>
+                </summary>
+            </section>
+            <section className="pb-4">
+                {body}
+            </section>
+            <aside className="text-sm font-semibold">
```

Comment:
I am not entirely sure `aside` is the best option here. `Aside` is intended to be used for indirectly related content, but these are calls to action for the user - to vote or comment. I often think of `aside` as a literal aside in a conversation. There may be an argument for using `aside` here, which I'm open to hearing, but for me it doesn't feel quite right